src/__tests__/interactions-helper.js

```diff
@@ -123,3 +123,58 @@ export const waitForElementVisible = getterFunc => {
 function logFunctionStart(name) {
   expect(` ➡️ ${name}`).toBeDefined();
 }
+
+function getElementClientCenter(element) {
+  const { left, top, width, height } = element.getBoundingClientRect();
+  return {
+    x: left + width / 2,
+    y: top + height / 2
+  };
+}
+
+const getCoords = ({ toElm, toCoords, delta, from }) => {
```

Comment:
we can split it but I don't want to move this logic to drag function or to consumer of drag function - so we'll still need this getCoords - witch can be spllited to several, but I don't see what it will give us.